src/foam/u2/view/RadioView.js

```diff
@@ -37,23 +54,34 @@ foam.CLASS({
   listeners: [
     function onChoicesUpdate() {
       var self = this;
+      var id;
 
       this.removeAllChildren();
 
       this.add(this.choices.map(function(c) {
         return this.E('div').
+          addClass('foam-u2-RadioContainer').
```

Comment:
I think it would be better to use the this.myClass() pattern here still, it will still be a name we can customize from outside, but then there' no risk of it accidentally conflicting with another class, especially for the Label class below.

The <label> element can already be addressed using  "^ label"